### DISCUSSION TITLE
Feature Request: a17-fieldset tag support a new attribute to open if contained fields are not empty

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1119] [Ideas]
When new fieldsets are created in twill such as....

```
<a17-fieldset title="Things to do" id="things" :open="false">
<!-- fields -->
</a17-fieldset>
```

this optional attribute :open can be set to true or false.    Could functionality be added for it also to have an auto value?   This would then open automatically if vue code evaluates it and determines that at least one of the fields within the fieldset has completed content.    If nothing has been completed it would stay closed.

**Comment by adam-jones-net** [comment]
So I am suggesting...

```
<a17-fieldset title="Things to do" id="things" :open="auto">
<!-- fields -->
</a17-fieldset>
```

**Comment by ifox** [comment]
We would definitely welcome a PR for this, but considering the form state doesn't have any knowledge about which fieldset holds which field, I think this would require something like selecting all inputs in the child DOM of a fieldset, and checking that none of them is present in the form's vuex state. I'm worried that some of our more complex fields do not necessarily use an HTML input in the DOM though.
